### ISSUE TITLE
Kernel updates?

### ISSUE THREAD

**Issue by dawidi** [sigboe/GPD-Arch-LUKS-LVM#18]
I set up my Pocket in January using archlinux-gpd-pocket-2018.01.05-x86_64.iso and your guide.

Updating with pacman -Syu every couple of weeks since, I've seen some minor system issues appear and disappear, but uname still shows `Arch Linux 4.15.0-rc6-jwrdegoede` so I'm assuming I'm still running a kernel from four months back.
Can I do anything to get a newer kernel? Or is this still the latest version with Pocket patches? I can't find a GPDPocket kernel repo at https://github.com/jwrdegoede, so where does it come from?

Basically, what's the longer-term strategy for keeping Arch fully up-to-date on the Pocket, after installing from the ISO?

**Comment by sigboe** [comment]
I've been to sick to work on the project lately, but here is the long term strategy https://aur.archlinux.org/packages/linux-jwrdegoede-git/

Possibly the repo this ISO installed for you can be updated to include that package instead of its package that is unable to be updated. @joshskidmore

**Comment by Thaodan** [comment]
I have an optimized version of linux-pf.
Upstream contains all needed fixes.
https://thaodan.de/home/bidar/home-thaodan/x86_64/

**Comment by Thaodan** [comment]
linux-pf-silvermont

**Comment by sigboe** [comment]
@Thaodan Have you made any optimizations that are relevant to the GPD Pocket?

**Comment by Thaodan** [comment]
I enabled all needed kernel flags that were missing from ARCH and pf and build the kernel with march silvermont (the soc of the device).
I also split the packages present In the repos to the minimum and wrote a script for non interactive install.

Am 16. Juli 2018 12:59:03 MESZ schrieb "Sigurd Bøe" <notifications@github.com>:
